requirements.txt

```diff
@@ -1,5 +1,5 @@
-https://codeload.github.com/pymedusa/adba/tar.gz/112c04b50d318802b4a0c7924f8e26aaf9e60a5b#egg=adba
-https://codeload.github.com/Diaoul/babelfish/tar.gz/f403000dd63092cfaaae80be9f309fd85c7f20c9#egg=babelfish
+adba @ https://codeload.github.com/pymedusa/adba/tar.gz/112c04b50d318802b4a0c7924f8e26aaf9e60a5b#egg=adba
```

Comment:
@sharkyhk our build is having issues with the "adba @ ...." Should I revert those changes?

Reply:
Oh are you talking about `security/snyk`?

Reply:
Yeah

Reply:
I didn't realize they couldn't read this format. I'll revert it.

Reply:
Nope, I guess it's something else, because it's still failing.